### ISSUE TITLE
package.json dependencies list does not contain all dependencies

### ISSUE THREAD

**Issue by ialarmedalien** [andreas-marschke/kotojs-charts-library#1]
Just cloned the repo from GitHub, ran `npm install` and then tried some of the tasks in the Gruntfile, but the following modules are missing from the `package.json` dependencies list: 
- `webpack`
- `webpack-dev-server`
- `grunt-contrib-cssmin`

Additionally, there are missing asset files that make two of the three uglify tasks fail; from the Gruntfile config, they are:

```
      dist: {
        files: {
          "dist/js/deps.min.js": [
            "assets/vendor/highlightjs/highlight.pack.js",
            "assets/vendor/jquery/dist/jquery.min.js",
            "assets/vendor/bootstrap/dist/js/bootstrap.js",
            "assets/vendor/d3/d3.min.js",
            "assets/vendor/d3-tip/index.js",
            "assets/vendor/koto/dist/koto.js"
          ]
        }
      },
```
and
```
      app: {
        files: {
          "dist/js/app.min.js": [
            "assets/js/app/*.js"
          ]
        }
      }
```

Presumably they were installed with bower (there is a `.bowerrc` file committed, but no `bower.json`), but excluded by the `.gitignore` file. It would be helpful to either add documentation on the extra packages that should be installed manually, or include a `bower.json` file or [migrate to yarn](https://bower.io/blog/2017/how-to-migrate-away-from-bower/).

**Comment by andreas-marschke** [comment]
Yeah I apologize I pushed an incomplete version. I'll update it when I have the time.
